models/Element/ValidationException.php

```diff
@@ -38,7 +38,7 @@ public function getSubItems()
     /**
      * @param array $subItems
      */
-    public function setSubItems($subItems)
+    public function setSubItems(array $subItems)
```

Comment:
that actually is a BC break, even though I could live with it. Still...

Reply:
You are right. In my opinion this method gets mostly used internally (in fact I just wondered why I got `null` when calling `getSubItems` although its method signature says `@return array`.
But if the core team sees any difficulties in adding the typehint we can first add a deprecation hint...

Reply:
Not sure about "mostly used internally" as the ValidationException class itself is mentioned in the docs.

https://pimcore.com/docs/5.x/Development_Documentation/Extending_Pimcore/Bundle_Developers_Guide/Plugin_Backend_UI.html

But yes, it is quite unlikely that the method is used quite often.

Reply:
I changed the code to only trigger a deprecation warning. We can introduce the typehint later...